src/DocumentFormat.OpenXml.Framework/Packaging/OpenXmlPartContainer.cs

```diff
@@ -1794,10 +1794,17 @@ internal OpenXmlPart CreateOpenXmlPart(string relationshipType)
         /// <param name="loadedParts">Temp collection to detect loaded (shared) parts.</param>
         internal void LoadReferencedPartsAndRelationships(OpenXmlPackage openXmlPackage, OpenXmlPart? sourcePart, RelationshipCollection relationshipCollection, Dictionary<Uri, OpenXmlPart> loadedParts)
         {
-            foreach (var relationship in relationshipCollection)
+            List<string> partsToIgnore = new()
             {
                 // Fix bug https://github.com/OfficeDev/Open-XML-SDK/issues/1205
-                if (relationship.RelationshipType == @"http://schemas.microsoft.com/office/2006/relationships/recovered")
+                @"http://schemas.openxmlformats.org/officeDocument/2006/relationships/calcChain",
```

Comment:
Is this something that is always missing? are there cases in which we wouldn't want to skip it?

Reply:
@twsouthwick  read the comments on #1279, the Excel team and I discussed this. For opening a package with the SDK, it's safe to skip this regardless of whether the relationship is valid (i.e. has a valid part) or not. Excel will always create the relationship and part as needed and doesn't rely on reading it.